src/ServiceControl/Operations/AuditImporter.cs

```diff
@@ -30,8 +30,12 @@ protected override void Setup(FeatureConfigurationContext context)
             {
                 context.Container.ConfigureComponent<AuditImporter>(DependencyLifecycle.SingleInstance);
                 context.Container.ConfigureComponent<AuditIngestor>(DependencyLifecycle.SingleInstance);
-
-                SetupImportFailuresHandler(context);
+                context.Container.ConfigureComponent(b =>
+                    new SatelliteImportFailuresHandler(b.Build<IDocumentStore>(), Path.Combine(b.Build<LoggingSettings>().LogPath, @"FailedImports\Audit"), msg => new FailedAuditImport
+                    {
+                        // TODO: Check if the usage of FailedTransportMessage breaks anything
```

Comment:
`FailedTransportMessage` will allow for proper header importing, I don't see how it would break anything

Reply:
Ok will remove the comments in both features